From b6bc74a12a6873dcb9e44a7c225f3fd611026f6a Mon Sep 17 00:00:00 2001
From: Siegfried-Angel Gevatter Pujals <rainct@ubuntu.com>
Date: Wed, 3 Feb 2010 00:35:36 +0100
Subject: [PATCH] Enable filename search-as-you-type

This enables search-as-you-type functionality in the main treeview,
which makes it much faster to use file-roller using only the keyboard.

https://bugzilla.gnome.org/show_bug.cgi?id=619738
---
 src/fr-window.c |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

Index: file-roller-3.3.92/src/fr-window.c
===================================================================
--- file-roller-3.3.92.orig/src/fr-window.c	2012-02-21 00:39:03.000000000 +0100
+++ file-roller-3.3.92/src/fr-window.c	2012-03-20 07:34:42.482360662 +0100
@@ -5612,8 +5612,6 @@
 
 	gtk_tree_view_set_rules_hint (GTK_TREE_VIEW (window->priv->list_view), TRUE);
 	add_file_list_columns (window, GTK_TREE_VIEW (window->priv->list_view));
-	gtk_tree_view_set_enable_search (GTK_TREE_VIEW (window->priv->list_view),
-					 FALSE);
 	gtk_tree_view_set_search_column (GTK_TREE_VIEW (window->priv->list_view),
 					 COLUMN_NAME);
 
